internal/server/headers.go

```diff
@@ -0,0 +1,26 @@
+package server
+
+import (
+	"fmt"
+	"strings"
+)
+
+func readHeaders(headers string, callback func(key string, value string)) error {
```

Comment:
just throw this into application.go, as its not complicated/big enough to deserve its own file